PocketCaddy/PocketCaddy/MainView/LaunchView.swift

```diff
@@ -4,27 +4,28 @@
 //
 //  Created by Byeon jinha on 2022/06/10.
 //
-
+import Combine
+import Foundation
 import SpriteKit
 import SwiftUI
 
 class MyTimer: ObservableObject {
-   var value: Int = 0
-   
+   @Published var value: Int = 0
+   @Published var goToHomeView = false
    init() {
-                                             //간격        //반복되기때문에 true   //timer을 in 해준다.
        Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { timer in
```

Comment:
```suggestion
       Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { [weak self] timer in
```

메모리 누수의 위험이 있어서, `weak self`가 추가되어야할 필요가 있어보여요 ~

Reply:
[weak self] 에 대해 공부해 오겠습니다.